ViAn/Video/shapes/shape.h

```diff
@@ -20,7 +20,8 @@ class Shape {
     static cv::Scalar qcolor_to_scalar(QColor col);
     static cv::Point qpoint_to_point(QPoint pnt);
 
-    static const int LINE_THICKNESS = 2;
+    static const int LINE_THICKNESS = 2; // Constant used for the thickness of the drawn shapes.
+    static constexpr double ALPHA = 0.6; // Costant used for the opacity.
 
 protected:
     cv::Scalar colour;
```

Comment:
Regarding the area, there is not much I can do about that since I'm using an OpenCV function for drawing polygons.
Regarding multiple area selections. I will not do that now. Maybe later if we have time and think it's an important thing to do.